sspl_test/run_qa_test.sh

```diff
@@ -23,12 +23,16 @@ IS_VIRTUAL=$(facter is_virtual)
 
 [[ $EUID -ne 0 ]] && sudo=sudo
 script_dir=$(dirname $0)
-source $script_dir/constants.sh
-SSPL_STORE_TYPE=${SSPL_STORE_TYPE:-consul}
+source "$script_dir"/constants.sh
+SSPL_STORE_TYPE=confstor
 
 plan=${1:-}
 avoid_rmq=${2:-}
 
+common_config=yaml:///etc/cortx/sample_global_cortx_config.yaml
+test_config=yaml:///opt/seagate/"$PRODUCT_FAMILY"/sspl/sspl_test/conf/sspl_tests.yaml
```

Comment:
sspl_tests.conf only we can keep, yaml being internal format(same as sspl.conf)

Reply:
done